srcpkgs/ReText/template

```diff
@@ -1,19 +1,24 @@
 # Template file for 'ReText'
 pkgname=ReText
-version=7.1.0
-revision=2
+version=7.2.1
+revision=1
 wrksrc="${pkgname,,}-${version}"
 build_style=python3-module
 hostmakedepends="ImageMagick python3 qt5-host-tools qt5-tools python3-setuptools"
 makedepends="python3-Markdown python3-Markups python3-PyQt5-webkit qt5-tools"
 depends="desktop-file-utils python3-chardet python3-docutils python3-Markdown
  python3-Markups python3-PyQt5-webkit"
+checkdepends="python3-pip"
```

Comment:
Fixing tests with pip means we are testing using packages from pypi instead of the void ones. Do `checkdepends="$depends"` instead and see where it fails. If it complains about trying to install pytest, add it as well.

Reply:
I get that, but I am not going to spend the time, sorry.

Reply:
Then don't add pip at all and allow the tests to be fixed in the future.

Reply:
Doing tests the right way causes python to abort in this case, which is why we want tests that actually run against the stuff we build rather than whatever wheels pip downloads.

Reply:
@ahesford @ericonr I just added a comments that tests fail with packages and succeed with pip. Thank you.